core/internal/processlib/processlib_linux.go

```diff
@@ -0,0 +1,26 @@
+package processlib
+
+import (
+	"os"
+	"syscall"
+)
+
+const (
+	// Linux syscall for prctl, see x/sys/unix SYS_PRCTL
+	PRCTL_SYSCALL = 157
+	// Set the parent-death signal of the calling process, see uapi/linux/prctl.h
+	PR_SET_PDEATHSIG = 1
+)
+
+func ShutdownOnParentExit(parentPid int) bool {
+	_, _, errno := syscall.Syscall(
+		uintptr(PRCTL_SYSCALL), uintptr(PR_SET_PDEATHSIG), uintptr(syscall.SIGKILL), 0)
```

Comment:
maybe make a const for the 0 or add a comment for that it suppose to be?

Reply:
it is an unused argument.   syscall convention is to just put 0s in those.